tgui/packages/tgui/interfaces/IFF.js

```diff
@@ -16,8 +16,6 @@ export const IFF = (props, context) => {
       <Window.Content scrollable>
         {!data.is_hackerman && (
           <Section title="ACCESS DENIED.">
-            <img src="https://wiki.baystation12.net/images/thumb/1/1f/Sol_Gov_Flag.png/300px-Sol_Gov_Flag.png" />
-            <br />
             {"FILE ACCESS ENCRYPTED."}
```

Comment:
```suggestion
            <img src="https://nsv.beestation13.com/mediawiki2/images/8/8f/SolGov.png" />
            <br />
            {"FILE ACCESS ENCRYPTED."}
```
We can replace it with our own wiki's flag file for now. Made the suggestion on the line below because you can't suggest on code that's removed 😔

Reply:
Aight, that works